gigboard-client/src/pages/Register/Register.jsx

```diff
@@ -1,27 +1,63 @@
+import { useState } from 'react';
+
+import { useHistory } from 'react-router-dom';
 import { Button, Form } from 'react-bootstrap';
 import 'bootstrap/dist/css/bootstrap.min.css'
 
+import Auth from '../../Models/Auth'
+
+
 const Register = () => {
+
+    const history = useHistory()
+    
+    const [ email, setEmail ] = useState('')
+    const [ password, setPasword ] = useState('')
+    const [ username, setUsername ] = useState('')
+
+    const [ message, setMessage ] = useState( null );
+    
+    const handleSubmit = async (e) => {
+        e.preventDefault()
+    
+        try {
+          const user = await Auth.register({ email, username, password });
+          console.log(user)
+        //   const token = user.data.signedJwt
+    
+        //   localStorage.setItem( 'uid', token );
+        //   props.setCurrentUser(token);
+    
+          history.push('/gigs');
+          
+        } catch (error) {
+          setMessage(error.response) 
+        }
+    
+      }
+    
+    console.log( message );
+    
     return (
         <>
         <div>Register</div>
```

Comment:
If we're going to need more information to create the user we'll need to add those inputs into the form (firstName, lastName, etc...)